docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -1,69 +1,32 @@
 ---
-title: Run E2E tests on EAS Build
+title: Run E2E tests on EAS Workflows
 sidebar_title: Run E2E tests
-description: Learn how to set up and run E2E tests on EAS Build with Maestro.
+description: Learn how to set up and run E2E tests on EAS Workflows with Maestro.
 ---
 
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+import { Grid01Icon } from '@expo/styleguide-icons/outline/Grid01Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
 import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
 import { Terminal } from '~/ui/components/Snippet';
 import { Step } from '~/ui/components/Step';
-import { VideoBoxLink } from '~/ui/components/VideoBoxLink';
-
-> **info** This guide will evolve over time as support for E2E testing in EAS Build improves. If you are looking for an archived guide on how to run E2E tests on EAS Build using Detox, you can see it in the [archive section](/archive/e2e-tests/).
 
-In this guide, you will learn how to create and run E2E tests on EAS Build using [Maestro](https://maestro.mobile.dev/), which is one of the most popular tools for running E2E tests in mobile apps.
+Learn how to run end-to-end (E2E) tests on EAS Workflows using [Maestro](https://maestro.mobile.dev/).
 
-The example demonstrates how to configure your EAS Build Maestro E2E tests workflow using the [default Expo template](/more/create-expo/#--template). For your own app, you will need to adjust the flows to match your app's UI.
-
-<VideoBoxLink videoId="-o-bfIRrg9U" title="Watch: How to run E2E tests on EAS Build" />
+[EAS Workflows](https://expo.dev/eas/workflows) is a new React Native CI/CD service for Android, iOS, and Web. See the [EAS Workflows documentation](/eas/workflows/get-started) for more information.
```

Comment:
```suggestion
```

Let's remove this. Instead, we should make this doc part of the EAS Workflows reference. I think this kind of info is more suitable for landing/overview page of a section.